Flask assertion error in tests_cli

```
=== start (solutions/python_flask_tests/pytest --app-config-path config.json)
runtest[start]: /usr/bin/timeout 120 /usr/bin/time -f %e --quiet -o /tmp/tmp.452zJhpLvm  /home/oeadmin/workspace/Mystikos/Standalone-Pipelines/Solutions-Tests-Pipeline/build/bin/myst exec-sgx rootfs /usr/local/bin/python /usr/local/bin/pytest --app-config-path config.json
mytikos: warn: exec.c(1145): myst_exec(): 
    The thread stack size may be too small for the given program interpreter
    (link loader), which could result in stack overflows. Consider changing
    the thread stack size to at least 1048576 bytes, using the --thread-stack-size
    option or the ThreadStackSize configuration setting.
    [interpreter=/lib64/ld-linux-x86-64.so.2]
    [program=/usr/local/bin/python]

============================= test session starts ==============================
platform linux -- Python 3.9.13, pytest-7.0.0, pluggy-1.0.0
rootdir: /app/flask, configfile: setup.cfg, testpaths: tests
collecting ... 
collecting 213 items                                                           
collected 481 items                                                            

tests/test_appctx.py ..............                                      [  2%]
tests/test_async.py ........                                             [  4%]
tests/test_basic.py .................................................... [ 15%]
........................................................................ [ 30%]
......                                                                   [ 31%]
tests/test_blueprints.py ............................................... [ 41%]
..........                                                               [ 43%]
tests/test_cli.py .........................................F............ [ 54%]
.                                                                        [ 54%]
tests/test_config.py ...................                                 [ 58%]
tests/test_converters.py ..                                              [ 59%]
tests/test_helpers.py ............ytikos: warn: exec.c(1145): myst_exec(): 
    The thread stack size may be too small for the given program interpreter
    (link loader), which could result in stack overflows. Consider changing
    the thread stack size to at least 1048576 bytes, using the --thread-stack-size
    option or the ThreadStackSize configuration setting.
    [interpreter=/lib64/ld-linux-x86-64.so.2]
    [program=/usr/local/bin/python]

m..................                     [ 65%]
tests/test_instance_config.py .XXX.......                                [ 67%]
tests/test_json.py .................................                     [ 74%]
tests/test_json_tag.py ..............                                    [ 77%]
tests/test_logging.py ......                                             [ 78%]
tests/test_regression.py .                                               [ 79%]
tests/test_reqctx.py ..............                                      [ 81%]
tests/test_session_interface.py .                                        [ 82%]
tests/test_signals.py .......                                            [ 83%]
tests/test_subclassing.py .                                              [ 83%]
tests/test_templating.py ...............................                 [ 90%]
tests/test_testing.py ..........................                         [ 95%]
tests/test_user_error_handler.py .........                               [ 97%]
tests/test_views.py ............                                         [100%]

=================================== FAILURES ===================================
_____________________________ TestRoutes.test_sort _____________________________

self = <test_cli.TestRoutes object at 0x12767aaf0>
invoke = functools.partial(<bound method CliRunner.invoke of <click.testing.CliRunner object at 0x1275f5250>>, <FlaskGroup None>)

    def test_sort(self, invoke):
        default_output = invoke(["routes"]).output
        endpoint_output = invoke(["routes", "-s", "endpoint"]).output
        assert default_output == endpoint_output
        self.expect_order(
            ["static", "yyy_get_post", "aaa_post"],
            invoke(["routes", "-s", "methods"]).output,
        )
        self.expect_order(
            ["yyy_get_post", "static", "aaa_post"],
            invoke(["routes", "-s", "rule"]).output,
        )
>       self.expect_order(
            ["aaa_post", "yyy_get_post", "static"],
            invoke(["routes", "-s", "match"]).output,
        )

tests/test_cli.py:480: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <test_cli.TestRoutes object at 0x12767aaf0>
order = ['aaa_post', 'yyy_get_post', 'static']
output = 'Endpoint      Methods    Rule\n------------  ---------  -------------------------\nstatic        GET        /static/<path:filename>\nyyy_get_post  GET, POST  /get_post/<int:x>/<int:y>\naaa_post      POST       /zzz_post\n'

    def expect_order(self, order, output):
        # skip the header and match the start of each row
        for expect, line in zip(order, output.splitlines()[2:]):
            # do this instead of startswith for nicer pytest output
>           assert line[: len(expect)] == expect
E           AssertionError: assert 'static  ' == 'aaa_post'
E             - aaa_post
E             + static

tests/test_cli.py:461: AssertionError
=========================== short test summary info ============================
FAILED tests/test_cli.py::TestRoutes::test_sort - AssertionError: assert 'sta...
=================== 1 failed, 477 passed, 3 xpassed in 3.75s ===================
runtest[finish]: /usr/bin/timeout 120 /usr/bin/time -f %e --quiet -o /tmp/tmp.452zJhpLvm  /home/oeadmin/workspace/Mystikos/Standalone-Pipelines/Solutions-Tests-Pipeline/build/bin/myst exec-sgx rootfs /usr/local/bin/python /usr/local/bin/pytest --app-config-path config.json
    failed (solutions/python_flask_tests/pytest --app-config-path config.json)
    16.78 seconds
```

--- a/solutions/python_flask_tests/Dockerfile
+++ b/solutions/python_flask_tests/Dockerfile
@@ -3,8 +3,8 @@ FROM python:3.9-slim
 WORKDIR /app
 
 RUN apt update && apt install -y git
-# Checkout v2.1.0
-RUN git clone https://github.com/pallets/flask && cd flask && git checkout 65b0eef303dfec6b7baa66ff34253e0285e1c3bf
+# Checkout Flask v2.1.3
+RUN git clone https://github.com/pallets/flask && cd flask && git checkout 187d7179f605d28c3d24e9f4d65d3295fb099afe
 
 WORKDIR /app/flask
 

